tests/Features/Testbench.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+function my_environment($app)
+{
+}
+
+test('it will add a define-env annotation', function () {
+    $phpDoc = (new ReflectionClass($this))->getMethod($this->name());
+    expect(str_contains($phpDoc->getDocComment(), '* @define-env my_environment'))->toBeTrue();
```

Comment:
We can simplify this test 🤷🏻
```suggestion
    expect($phpDoc->getDocComment())->toContain('* @define-env my_environment');
```